website/docs/d/subnet_ids.html.markdown

```diff
@@ -10,24 +10,24 @@ description: |-
 
 `aws_subnet_ids` provides a list of ids for a vpc_id
 
-This resource can be useful for getting back a list of subnet ids for a vpc.
+This resource can be useful for getting back a set of subnet ids for a vpc.
 
 ## Example Usage
 
 The following shows outputing all cidr blocks for every subnet id in a vpc.
 
 ```hcl
-data "aws_subnet_ids" "example" {
-  vpc_id = "${var.vpc_id}"
+data aws_subnet_ids example {
+  vpc_id = var.vpc_id
 }
 
-data "aws_subnet" "example" {
-  count = "${length(data.aws_subnet_ids.example.ids)}"
-  id    = "${data.aws_subnet_ids.example.ids[count.index]}"
+data aws_subnet example {
+  count = length(data.aws_subnet_ids.example.ids)
+  id    = tolist(data.aws_subnet_ids.example.ids)[count.index]
 }
 
-output "subnet_cidr_blocks" {
-  value = "${data.aws_subnet.example.*.cidr_block}"
+output subnet_cidr_blocks {
+  value = data.aws_subnet.example.*.cidr_block
```

Comment:
This should be replaced with `[for s in data.aws_subnet.for-each-example : s.cidr_block]` when updated to use `for_each`